CHANGELOG.md

```diff
@@ -9,6 +9,11 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 * Update pipeline template to nf-core/tools `1.12`
 
+### Software dependencies
+| Dependency              | Old version | New version |
```

Comment:
```suggestion

| Dependency              | Old version | New version |
```

Reply:
Can we add a space here too please? Thanks. Feel free to merge when the tests pass 👍